faker_edu/__init__.py

```diff
@@ -136,6 +136,31 @@
     'Teaching Assistant',
 ]
 
+COLLEGETYPES = [
+    'Arts & Sciences',
+    'Literature, Science, & the Arts',
+    'Engineering',
+    'Business',
+    'Commerce',
+    'Medicine',
+    'Information',
+    'Kinesiology',
+    'Nursing',
+    'Law',
+    'Public Policy',
+    'Public Health',
+    'Architecture',
+    'Art & Design',
+    'Music',
+    'Theatre',
+    'Nursing',
```

Comment:
Nursing appears twice on this list.  Once you sort them it should be easy to find and remove all duplicates.